Jenkinsfile

```diff
@@ -0,0 +1,145 @@
+#!groovy
+//
+// Jenkinsfile to control automated testing using https://jenkins.tjhei.info
+//
+// To run the tests locally, you can execute the following in a burnman working directory:
+//
+// docker run --rm -v "$(pwd):/src" tjhei/burnman:v4 /bin/bash -c "cd /src && ./test.sh"
```

Comment:
It might be worth adding that the user must first install docker and add themselves as a user.
`sudo apt install docker.io`
`sudo usermod -a -G docker $USER`

It's also definitely worth commenting that attempting to run the tester on a local machine will involve downloading 2 Gb of files...